src/components/ui/OTP/OTP.tsx

```
@@ -0,0 +1,85 @@
+      className={`${rootClass} otp-${size} flex space-x-2 ${className}`}
+      data-accent-color={color ?? undefined}
+    >
+      {otp.map((digit, index) => (
+        <input
+          key={index}
+          type="text"
+          value={digit}
+          onChange={(e) => handleChange(e.target.value.slice(0, 1), index)}
+          onKeyDown={(e) => handleKeyDown(e, index)}
+          onPaste={handlePaste}
+          ref={(el) => (inputsRef.current[index] = el)}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)